### PR TITLE
session02 exercises

### PR DESCRIPTION
**Pull request by habtamua**
- printer_Grid_Ex.py (__modified_ and _completed__)
- fizzbuzz.py ( _completed__)
- series.py (not _completed_)_

Haven't done yet for fibonacci series exercise (series.py) and am still working on it.
Thank you

### CODE REVIEW

File: students/HABTAMU/session02/printer_Grid_Ex.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                print("+", end=" ")            
+            elif ((i in range(0,n+5,5) and j > 0 and j < n)) :
+                print("-", end=" ")            
+            elif ((j in range(0,n+5,5) and i > 0 and i < n)) :
+                print("|", end="     ")           
+            else :
+                print("", end=" ")
+        print()
+n = 25
+print_rectangle(n)
```
Comment (charlesrobison) [inline students/HABTAMU/session02/printer_Grid_Ex.py]: Appreciate the efforts here - I'll go ahead and merge this, but feel free to resubmit if you want to progress further.

File: students/HABTAMU/session02/series.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    if n <= 0:
+        return 1
+    else:
+        for i in range(n):
+            print(a, end=" ")
+
+            c = a + b
+            a = b
+            b = c
```
Comment (charlesrobison) [inline students/HABTAMU/session02/series.py]: Interesting solution!  The assignment only required the return of the nth value, but this works.

File: students/HABTAMU/session02/series.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+#     elif n == 1:
+#         x = {0,1}
+#         return 1
+#     else :
+# #        for i in range(n):
+#         n = fibonacci(n-2) + fibonacci(n-1)
+#         x.append('n')
+#         return x
+#         # x = {0,1,i}
+#         # print(x)
```
Comment (charlesrobison) [inline students/HABTAMU/session02/series.py]: Looks like you're still working on the lucas and sum_series portions.  I'll leave this open so you can complete.

File: students/HABTAMU/session02/fizzbuzz.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+for i in range(100):
+    if (i%3 == 0) :
+        if (i%5 == 0):
+            print(i, "FizzBuzz")
+        else:
+            print(i, "Fizz")
+    elif (i%5 == 0):
+        print(i,"Buzz")
+    else:
+        print(i)
```
[the remark this answers is not in the document]
Comment (habtamua) [reply students/HABTAMU/session02/fizzbuzz.py]: i made that change as (101) to from 0 to 100...

Thank you

**Comment by PythonCHB** [comment]
merging -- still need sum_series --

**Pull request merged**